src/main/java/edu/kis/powp/jobs2d/command/letters/FCommandLetter.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.command.letters;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class FCommandLetter implements ComplexCommand {
+
+    private Job2dDriver job2dDriver;
+
+    public FCommandLetter(Job2dDriver job2dDriver) {
+        this.job2dDriver = job2dDriver;
+    }
+
+    @Override
+    public void execute() {
+        job2dDriver.setPosition(0, 0);
```

Comment:
Minor issue: coords are fixed. It would be nicer to have some params responsible for size and shift.